package.json

```diff
@@ -40,7 +40,6 @@
     "gh-pages": "3.2.3",
     "is-interactive": "2.0.0",
     "lerna-changelog": "1.0.1",
-    "netlify": "6.1.29",
```

Comment:
Removed since deployments relied on a single API from this package that has now been deprecated. Deployments moving forward will rely on [`netilify-cli`](https://www.npmjs.com/package/netlify-cli)'s deploy command instead of [`netlify`](https://www.npmjs.com/package/netlify)'s deprecated `.deploy()` method.

[Source: Netlify v7 breaking changes](https://github.com/netlify/js-client/releases/tag/v7.0.0)